include/NovelRT/Maths/QuadTree.h

```diff
@@ -37,14 +38,15 @@ namespace NovelRT::Maths {
       return _parent;
     }
 
-    const GeoBounds& getBounds() const noexcept {
+    GeoBounds getBounds() const noexcept {
       return _bounds;
     }
 
     const std::shared_ptr<QuadTreePoint>& getPoint(size_t index) const noexcept {
       return _points[index];
     }
 
+    //TODO: Im fairly certain this can't exist in the C API? Figure this out
```

Comment:
Was this solved?

Reply:
Yes - we don't port the templated versions over because we cannot specify every type. Only in certain instances do we vouch to port this over - see Ink for example.

Reply:
I'll remove the comment